doc/source/admin/galaxy_options.rst

```diff
@@ -5764,7 +5770,11 @@
 
 :Description:
     Install repository dependencies when installing tools from the
-    Tool Shed.
+    Tool Shed. These are other Tool Shed repositories that the tool
```

Comment:
I think this might be the option that is required to install tool suits in which case one should not disable this?

Reply:
You're correct (tested this). Updated the description.